src/core/tests/components/test_repeat.py

```diff
@@ -0,0 +1,49 @@
+import pytest
```

Comment:
Could you add some docstrings to the functions to aid debugging if we break something